src/main/java/com/glencoesoftware/omero/ms/core/RedisCacheVerticle.java

```diff
@@ -78,10 +77,17 @@ public void start() {
                 });
     }
 
+    @Override
+    public void stop() {
+        connection.close();
+        client.shutdown();
```

Comment:
If we're doing connection close and client shutdown here we probably want to do the exact same thing in `OmeroWebRedisSessionStore` right?